src/components/AsideHeader/components/FirstPanel.tsx

```diff
@@ -42,53 +44,58 @@ export const FirstPanel = React.forwardRef<HTMLDivElement>((_props, ref) => {
     }, [ref]);
 
     const isExpandedByHover = compact && isExpanded;
-
     return (
         <React.Fragment>
-            <div
-                className={b('aside', {'expanded-by-hover': isExpandedByHover}, className)}
-                style={{width: size}}
-                data-qa={qa}
-                onMouseEnter={onExpand}
-                onMouseLeave={onFold}
+            <CSSTransition
```

Comment:
The `CSSTransition` components don't use `unmountOnExit` or `mountOnEnter`. Without these props, the transition classes remain applied even after exit. Is it intentional?